src/pages/Article/ArticlePage.jsx

```diff
@@ -33,37 +19,37 @@ function ArticlePage({ article }) {
           justifyContent: 'center',
           alignItems: 'center',
           height: 'calc(100% - 60px)',
-          overflowY: 'auto',
-          backgroundColor: '#FF0000',
+          overflowY: 'auto'
         }}
       >
         <Box
           sx={{
             height: '80%',
-            width: '40%',
+            width: '60%',
             display: 'flex',
+            flexDirection: 'column',
             justifyContent: 'center',
             alignItems: 'center',
             backgroundColor: '#D9D9D9',
```

Comment:
Please use theme data for this color

Reply:
the theme data has 4 colour, 3  of the colour is blue with different tone and the 4th one is contrastText which is just white

so am confused on the colour you want me to use, please explain better.

Reply:
This is how it looks with  'light' from theme data.... is that okay for you?
<img width="1349" alt="Screenshot 2023-11-13 at 11 20 40" src="https://github.com/SE2-WS23/Skillverse-Web/assets/104029083/cec9560a-1b0d-4e22-9cc8-5b69a4da9f20">

Reply:
@Chrissydrx @julix14 Uzoma seems to have worked on all of his points. 
Is this done now?

Reply:
No, he has to resolve the Code_Quality Issues

Reply:
And his page throws many errors in the browser console
<img width="886" alt="image" src="https://github.com/SE2-WS23/Skillverse-Web/assets/65715000/202456ef-db56-40f7-a9e5-981192eb8303">